src/index.html

```diff
@@ -35,17 +36,14 @@
     </style>
   </head>
   <body style="margin:0;">
-    <div id="root"></div> 
+    <div id="root"></div>
     <script>
-      if (typeof window !== 'undefined' &&
-        typeof navigator !== 'undefined' &&
-        navigator.serviceWorker) {
+      if (typeof window !== 'undefined' && typeof navigator !== 'undefined' && navigator.serviceWorker) {
         navigator.serviceWorker.register('/sw.js').then(function(reg) {
           reg.onupdatefound = function() {
             const installingWorker = reg.installing;
             installingWorker.onstatechange = function() {
-              if (installingWorker.state === 'installed' &&
-                  navigator.serviceWorker.controller) {
+              if (installingWorker.state === 'installed' && navigator.serviceWorker.controller) {
```

Comment:
can you please avoid unrelated changes to the PR? It adds noise to the review. Maybe create another PR for this?